test/instrumentation/modules/cassandra-driver/cassandra-driver.test.js

```diff
@@ -41,9 +41,14 @@ const testFixtures = [
     },
```

Comment:
The `name: 'cassandra-driver simple usage for versions <4.7.0',` line above should change to drop the "for versions <4.7.0"